tools/aws_lab_setup/sample-users.yml

```diff
@@ -2,3 +2,6 @@ users:
   - name: Luke Cage
     username: lcage
     email: lcage@marvel.com
+  - name: Andrius Beno
+    username: ab
+    email: andriusb@redhat.com
```

Comment:
Please no real people and their email addresses. Word on the street is that @abenokraitis is not a fan of spam. ;)